tests/test-core/jqUnit/js/jqUnit.js

```diff
@@ -172,11 +172,19 @@ var jqUnit = jqUnit || {};
         },
 
         assertDeepEq: function (msg, expected, actual) {
-            QUnit.propEqual(actual, expected, processMessage(msg));
+            if (fluid.isPrimitive(expected)) {
```

Comment:
Shall `actual` be verified as `isPrimitive` too? Otherwise, tests that have `expected` as an object but `actual` as a primitive would pass while they should actually fail. An example:

```
jqUnit.assertDeepEq("True is not deep equal to an object", {}, true);
```

The same comment for line 183.